frontend/public/components/filter-toolbar.tsx

```diff
@@ -258,6 +261,20 @@ const FilterToolbar_: React.FC<FilterToolbarProps & RouteComponentProps> = (prop
 
   const switchFilter = (type: FilterType) => {
     setFilterType(FilterType[type]);
+    if (FilterType[type] === 'Name') {
```

Comment:
This is probably better as a switch statement.

Reply:
done